lib/segment/src/index/field_index/map_index/mutable_map_index.rs

```diff
@@ -14,15 +14,15 @@ use crate::entry::entry_point::{OperationError, OperationResult};
 use crate::types::PointOffsetType;
 
 pub struct MutableMapIndex<N: Hash + Eq + Clone + Display + FromStr> {
-    map: HashMap<N, BTreeSet<PointOffsetType>>,
-    point_to_values: Vec<Vec<N>>,
+    pub(crate) map: HashMap<N, BTreeSet<PointOffsetType>>,
+    pub(crate) point_to_values: Vec<Vec<N>>,
     /// Amount of point which have at least one indexed payload value
-    indexed_points: usize,
-    values_count: usize,
-    db_wrapper: DatabaseColumnWrapper,
+    pub(crate) indexed_points: usize,
+    pub(crate) values_count: usize,
+    pub(crate) db_wrapper: DatabaseColumnWrapper,
```

Comment:
Thanks for the proposal. `pub(super)` is really better because I opened visibility only to avoid loading code duplication - loading from DB is shared between mutable and immutable state (immutable state creates mutable one, loads from DB and does conversion)